modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/checkpointing/CheckpointCommand.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2022 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.commandline.checkpointing;
+
+import java.util.logging.Logger;
+import org.apache.ignite.internal.client.GridClientConfiguration;
+import org.apache.ignite.internal.commandline.AbstractCommand;
+import org.apache.ignite.internal.commandline.Command;
+import org.apache.ignite.internal.commandline.CommandArgIterator;
+
+import static org.apache.ignite.internal.commandline.Command.usage;
+import static org.apache.ignite.internal.commandline.CommandList.CHECKPOINT;
+import static org.apache.ignite.internal.commandline.checkpointing.CheckpointingSubCommandsList.FORCE;
+
+public class CheckpointCommand extends AbstractCommand<Object> {
+    /**
+     *
+     */
+    private Command<?> delegate;
+
+    /** {@inheritDoc} */
+    @Override public void printUsage(Logger log) {
+        usage(log, "Start checkpointing process:",
+            CHECKPOINT
+        );
+    }
+
+    /** {@inheritDoc} */
+    @Override public String name() {
+        return CHECKPOINT.toCommandName();
+    }
+
+    @Override public void parseArguments(CommandArgIterator argIter) {
+        if (!argIter.hasNextArg()) {
```

Comment:
It seems that this code is not quite correct. In case of using auto-confirmation syntax: 'control.sh --checkpoint --yes`  this code throws an exception(Expected checkpointing action).

Moreover, I don't think that we need `CheckpointForceCommand` at all.